4.4/4.4.4/inadex.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 4.4.4
+Создать объект со следующей логикой:
+Появляется вопрос: "Введите любое число".
+Потом появляется алерт с сообщением: "Х - [четное; нечетное] число"
+<script>
+	const newObj = {
+		getNum() {
```

Comment:
it would be better if you define the property `number` with the value `null` at the beginning in order to see easily all properties (methods) of this object. If my explanation is not very clear, please call me )